### ISSUE TITLE
[Question] Not able to start firefox after installation

### ISSUE THREAD

**Issue by rai-gaurav** [microsoft/playwright#9027]
I am facing issue with the startup of firefox browser.
I am trying to run the basic example present at - https://playwright.dev/python/docs/intro/#usage
The browser get's launched but it stops there and doesn't open the given link.
![firefox_failure](https://user-images.githubusercontent.com/23376158/134023182-aa7fef3c-d895-4ef9-9a4b-c501ddb3d36d.PNG)

The browser installation is successfull (Followed pip installtion at- https://playwright.dev/python/docs/intro/#installation).
I tried to open the firefox nightly browser manually(present at - %USERPROFILE%\AppData\Local\ms-playwright\firefox-1281\firefox) and able to open and browse the internet but not able to acheive the same through playwright code.

`set DEBUG= pw:browser` output shows some error

```
  pw:browser <launching> C:\Users\Rai\AppData\Local\ms-playwright\firefox-1281\firefox\firefox.exe -no-remote -wait-for-browser -foreground -profile C:\Users\Rai\AppData\Local\Temp\playwright_firefoxdev_profile-A39Le9 -juggler-pipe --ignore-certificate-errors -silent +0ms
  pw:browser <launched> pid=9672 +8ms
  pw:browser [pid=9672][out]  +9s
  pw:browser [pid=9672][out] Juggler listening to the pipe +1ms
  pw:browser [pid=9672][out] console.warn: SearchSettings: "get: No settings file exists, new profile?" (new NotFoundError("Could not open the file at C:\\Users\\Rai\\AppData\\Local\\Temp\\playwright_firefoxdev_profile-A39Le9\\search.json.mozlz4", (void 0))) +624ms
  pw:browser [pid=9672][out] Crash Annotation GraphicsCriticalError: |[0][GFX1-]: RenderCompositorSWGL failed mapping default framebuffer, no dt (t=80.8959) [GFX1-]: RenderCompositorSWGL failed mapping default framebuffer, no dt +2
```
Do I need some extra library or dependency which is missing from my machine ? Since nothing is mentioned in doc, looks like this is something specific to me. But I am not able to figure out which dependency is missing.

Playwright version - 1.14.1
OS - Windows 10 (64 bit)
Firefox nightly - 91.0

I am able to test the same code in chrome, edge and webkit except firefox.
Please let me know if you need any other details.
Also I am using the Python(3.8.4) flavour. But I tried to test a small piece of code in Node.js also and got the same issue for firefox, hence created the issue here.

**Comment by mxschmitt** [comment]
@rai-gaurav are you on some cooperate environment? With version 1.15.0 something related to that should be fixed. Would be awesome if you could verify if this version behaves the same for you.

**Comment by rai-gaurav** [comment]
@mxschmitt Forgot to mentioned(sorry about that), yes I am inside corporate network behind proxy.
I followed [this](https://playwright.dev/python/docs/browsers#install-behind-a-firewall-or-a-proxy) for installation.
I upgraded playwright to 1.15.0 and downgraded to 1.13.1 also, installed the browsers, but in both cases I got the same issue.
All the browsers are working except firefox.

I can access - `https://playwright.azureedge.net` and subsequent path from browser and download zip file.
